libindy_vdr/include/libindy_vdr.h

```diff
@@ -0,0 +1,265 @@
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+
+#define DEFAULT_ACK_TIMEOUT 20
+
+#define DEFAULT_CONN_ACTIVE_TIMEOUT 5
+
+#define DEFAULT_CONN_REQUEST_LIMIT 5
+
+#define DEFAULT_FRESHNESS_TIMEOUT 300
+
+#define DEFAULT_REPLY_TIMEOUT 60
+
+#define DEFAULT_REQUEST_READ_NODES 2
+
+enum ErrorCode
+#ifdef __cplusplus
+  : int64_t
+#endif // __cplusplus
+ {
+  Success = 0,
+  Config = 1,
+  Connection = 2,
+  FileSystem = 3,
+  Input = 4,
+  Resource = 5,
+  Unavailable = 6,
+  Unexpected = 7,
+  Incompatible = 8,
+  PoolNoConsensus = 30,
+  PoolRequestFailed = 31,
+  PoolTimeout = 32,
+};
+#ifndef __cplusplus
+typedef int64_t ErrorCode;
+#endif // __cplusplus
+
+typedef int64_t CallbackId;
+typedef int64_t PoolHandle;
+typedef int64_t RequestHandle;
+typedef const char *FfiStr;
```

Comment:
Regenerating the cbindgen seems to remove this diff:

```diff
@@ -40,14 +40,6 @@ typedef int64_t ErrorCode;
 #endif // __cplusplus

 typedef int64_t CallbackId;
-typedef int64_t PoolHandle;
-typedef int64_t RequestHandle;
-typedef const char *FfiStr;
-
-typedef struct {
-    int64_t len;
-    uint8_t *data; // note: nullable
-} ByteBuffer;

 #ifdef __cplusplus
 extern "C" {
```

adding this to `cbindgen.toml` will add `PoolHandle` and `RequestHandle` to the generation. 

```toml
after_includes = """
typedef int64_t PoolHandle;
typedef int64_t RequestHandle;
"""
```

and adding this would include the ByteBuffer and FfiStr:

```toml
[parse]
parse_deps = true
include = ["ffi-support"]
```

Reply:
Thanks, yes the parsing of the `ffi_support` crate wasn't working at all for me before, but it is now, so go figure. I adjusted the code generation for the handle types so manual additions aren't required anymore.